README.md

```diff
@@ -1,16 +1,189 @@
-# Title
+# Ruby Method Arguments
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe the functionality of arguments
+- Define methods that accept arguments
+- Recognize the difference between arguments and parameters
 
 ## Introduction
 
-## SWBAT 1
+In the real world, many interactions are customized or specific to our
+day-to-day experiences. For example, if you go to a coffee shop, they may take
+down your name to call up your order when it's ready. If visit a doctor's
+office, they may call your name when it's your turn to be seen. Variables that
+we experience in real life often need to be translated into to programs.
 
-## SWBAT 2
+In the example of the doctor's office or the coffee shop they have a "template"
+behavior that they apply to everyone, but that they change only slightly
+depending on the particular customer or patient. Methods can also be written to
+have a "template" of shared activity but which can be changed just a little bit
+depending on some particular data. The particular bits that make the change are
+called "arguments."
```

Comment:
I think these two paras can be blended to be stronger:

>>>
In the real world, many interactions have a "template"
behavior that they apply to everyone, but that they change only slightly
depending on a tiny change. For example, if you go to a coffee shop, they may take
down your name to call up your order when it's ready. If visit a doctor's
office, they may call your name when it's your turn to be seen. Methods can also be written to have a "template" of shared activity but which can be changed just a little bit
depending on some particular data. The particular bits that make the change are
called "arguments."

Same sentences, less text, stronger.

Reply:
Also might add **emphasis** around **your** to show that that's the part that's changing within the template.